sample-apps/react-native/dogfood/src/utils/setPushConfig.ts

```diff
@@ -41,14 +41,15 @@ export function setPushConfig() {
  * This is used to create a video client for incoming calls in the background on a push notification.
  */
 const createStreamVideoClient = async () => {
-  const userName = mmkvStorage.getString('username');
+  const userId = mmkvStorage.getString('userId');
+  const userName = mmkvStorage.getString('userName');
   const userImageUrl = mmkvStorage.getString('userImageUrl');
-  if (!userName || !userImageUrl) {
+  if (!userId || !userImageUrl) {
```

Comment:
lets add username to be mandatory too

Reply:
For custom users, we don't send `userName`, and honestly, it doesn't make sense to send userId as userName to me. Do you think it will add value?

Reply:
yes it will add value.. 

then a custom user has an empty username if in any other DF apps... they show userName in the participant view